ui/src/Components/Company/Company.js

```diff
@@ -1,7 +1,175 @@
+
+            let model = companyModel[index]
+            const url = SERVER_URL + "/" + model.id
+
+            axios.delete(url).then(response => response.data).catch(error => alert("error occured"))
+
+
+        }
+
+        setcompanyModels(prevmodels => {
+            prevmodels.splice(index, 1);
+            return prevmodels.slice()
+        });
+
+    }
+
+
+
+    return (
+        <div className="border shadow  p-4 m-4">
+
+            <h4>Company data</h4>
+            <div className="shadow border  p-3 rounded maindata w-100">
+                <div>
+                    <Link to="/Home" className="text-black"><FiAirplay className="icondata me-2" /> Home</Link>/
+                    <Link to="/Company" className="text-black">Company</Link>
+
+                </div>
+                <div>
+                    <b>Companies</b>
+                </div>
+
+            </div>
+
+            <div className="maindata  w-100 mt-2">
+                <div className="btn-group " role="group" aria-label="Basic radio toggle button group">
+                    <button className="btn btn-primary w" for="btnradio1">All</button>
+                    <button className="btn btn-outline-primary w shadow" for="btnradio1">Theams</button>
+                    <button className="btn btn-outline-primary w shadow" for="btnradio1">Companies</button>
+
+                </div>
+
+                <div>
+
+                    <button className="btn btn-success">
+                        <IoAddSharp className="text-white " />Add Company
+                    </button>
+                </div>
+
+            </div>
+
+            <div className="data w-100 mt-2">
+                <div className="shadow border  w-75 rounded widthinc">
+                    <b>Total:{Company.length}</b>
+
+                </div>
+
+                <div className="btn-group buttonClass" role="group" aria-label="Basic radio toggle button group">
+                    <button className={data == 'Card' ? 'btn btn-primary   ':'btn btn-outline-primary w '}>
+                        <BsFillGridFill className="icondata text-black" onClick={() => { setData("Card") }} />
+
+
+                    </button>
+                    <button className={data == 'Table' ? "btn btn-primary w shadow ms-2 ":'btn btn-outline-primary  ms-2'} for="btnradio1" onClick={() => { setData("Table") }}><FaListUl className="icondata text-black"  /> </button>
+                </div>
+
+            </div>
+            <div className={data == 'Card' ? 'd-block  ':'d-none'}>
+            {
+                data == 'Card' ? <Card /> : null
+            }
+            </div>
+           
+            <div className={data == 'Table' ? 'd-block tabledata mt-3 ':'d-none'}>
+                <table className="table table-striped table-hover table-bordered border-primary">
+                    <tbody>
+                        <tr>
+                            <th>companyName</th>
+                            <th>mobileNumber</th>
```

Comment:
Change the columns as below

1. Mobile no
2. Address1
3. Address2
4. Logoname
5. Status